Decisions.Equifax/ConsumerCreditReport/ConsumerCreditReportSteps.cs

```diff
@@ -26,137 +16,10 @@ public class ConsumerCreditReportSteps
-        }
-        
-        
-        
-        /// <summary>
-        /// Helper: Generic Get OAuth via client credentials
-        /// </summary>
-        private static string GetOAuthToken(string scope)
-        {
-            string requestUrl = ModuleSettingsAccessor<EquifaxSettings>.Instance.EquifaxOAuthUrl;
-            string clientId = ModuleSettingsAccessor<EquifaxSettings>.Instance.EquifaxClientId;
-            string clientSecret = ModuleSettingsAccessor<EquifaxSettings>.Instance.EquifaxClientSecret;
-
-            if (string.IsNullOrWhiteSpace(requestUrl) || string.IsNullOrWhiteSpace(clientId) ||
-                string.IsNullOrWhiteSpace(clientSecret))
-            {
-                log.Error(EquifaxConstants.SETTINGS_CONFIGURATION_EXCEPTION);
-            }
-            
-            
-            HttpWebRequest request = (HttpWebRequest) WebRequest.Create(requestUrl);
-            string body = "scope=" + System.Web.HttpUtility.UrlEncode( scope ) + "&grant_type=client_credentials";
-            string encoded = System.Convert.ToBase64String(System.Text.Encoding.GetEncoding("ISO-8859-1").GetBytes($"{clientId}:{clientSecret}"));
-            request.Method = "POST";
-            request.ContentLength = body.Length;
-            request.Headers.Add("Authorization", "Basic " + encoded);
-            request.Headers.Add("Content-Type", "application/x-www-form-urlencoded");
-
-            log.Debug($"URL:{requestUrl}\r\nHasClientId:{(!string.IsNullOrWhiteSpace(clientId))}\r\nHasClientSecret:{(!string.IsNullOrWhiteSpace(clientSecret))}\r\nHasAuthorization:{(!string.IsNullOrWhiteSpace(encoded))}");
-
-            // Write body
-            using (StreamWriter sw = new StreamWriter(request.GetRequestStream()))
-            {
-                sw.Write(body);
-                sw.Flush();
-            }
-            
-            // Get response
-            string responseString = string.Empty;
-            try
-            {
-                using (var httpResponse = (HttpWebResponse) request.GetResponse())
-                using (Stream dataStream = httpResponse.GetResponseStream())
-                {
-                    if (dataStream != null)
-                    {
-                        string sts = httpResponse.ContentEncoding;
-                        using (StreamReader streamReader = new StreamReader(dataStream))
-                        {
-                            responseString = streamReader.ReadToEnd();
-                        }
-                    }
-                }
-            }
-            catch (WebException ex)
-            {
-                log.Error(ex.Message);
-            }
-
-            OAuthResponse resp = JsonConvert.DeserializeObject<OAuthResponse>(responseString);
-
-            if (resp == null)
-            {
-                log.Error(EquifaxConstants.CANNOT_GET_OAUTH_TOKEN);
-            }
-            return resp.AccessToken;
+            string stepCalled = "LimitCreditReport";
```

Comment:
This variable is unused. Is this string important?

Reply:
not at all, just leftovers from a previous version I forgot to remove